app/lib/actions/pulp3/content_migration.rb

```diff
@@ -5,6 +5,7 @@ class ContentMigration < Pulp3::AbstractAsyncTask
 
       def plan(smart_proxy, options)
         sequence do
+          plan_action(Actions::Katello::OrphanCleanup::RemoveOrphans, smart_proxy)
```

Comment:
I don't think we should do the entire orphan cleanup, we really just need to remove already cleaned up units (that aren't in pulp2 anymore):

https://github.com/Katello/katello/blob/1ff3b402b2022f6cbced03fe82d790a989fb896e/app/services/katello/pulp3/migration_switchover.rb#L28

Reply:
@jlsherrill I've changed it to simply use the method you linked to, what do you think of that?